resources/docker/Dockerfile.linux

```diff
@@ -11,20 +11,18 @@
 
 FROM ubuntu:20.04
 
-ENV CONNEXTDDS_ARCH="x64Linux4gcc7.3.0"
+ARG USER_NAME=jenkins
+ARG USER_UID
+
 ENV DEBIAN_FRONTEND="noninteractive"
+ENV CONNEXTDDS_ARCH="x64Linux4gcc7.3.0"
 
 RUN apt-get update && apt-get install -y \
-    gcc \
```

Comment:
Isn't the multiline better? https://docs.docker.com/develop/develop-images/instructions/#apt-get

Reply:
Sure, why not?